storage/series/series.go

```diff
@@ -440,6 +442,31 @@ func (s *dbSeries) mergeBlock(
 	return nil
 }
 
+func (s *dbSeries) Update(blk block.DatabaseBlock) error {
+	s.Lock()
+	defer s.Unlock()
+
+	// ensure series is bootstraped before allowing any updates
+	if s.bs != bootstrapped {
+		return errSeriesNotBootstrapped
+	}
+
+	// we only accept updates to non-buffered data
+	// ensure buffer is past provided time
+	min, _ := s.buffer.MinMax()
+	if !blk.StartTime().Before(min) {
```

Comment:
Yep, intentionally so we can avoid repairing anything 'active' that will require an explicit buffer `Drain()`.

Is the concern that we'll wait too long before we repair the past 2 hour block?